bin/additional_tests/test_launch_interpret_json.py

```diff
@@ -0,0 +1,128 @@
+import unittest
+from bin.launch_interpret_json import interpret_json
+
+""" 
+to run this test you need to put a relatibve inmport in the JsonSchema import line in launch_interpret_json
```

Comment:
i don't think you need this comment, since every test is doing this import from bin

Reply:
expanded the comment.